automated/linux/widevine/widevine_unittest.yaml

```diff
@@ -0,0 +1,31 @@
+metadata:
+    name: widevine_unittest
+    format: "Lava-Test-Shell Test Definition 1.0"
+    description: "Widevine ce_cdm unit test"
```

Comment:
Please make the description more 'descriptive'. It is displayed in the documentation so oneliners are not a good idea.